/bridge_rbcar_controller_node:
  ros__parameters:
    sub_topic: "command"
    ros_1_server_ip: "localhost"
    ros_1_server_port: 8888
    ros_2_server_port: 8889
    Hz: 25